### PR TITLE
an/SKY-69: Delete Button in Post-Edit HTML

### PR DESCRIPTION
**Pull request by anurmamedov**
Delete button changed from bin to 'delete' written type.

wide size 
image

medium size
image

small size 
image

### CODE REVIEW

File: templates/post-edit.html [hunk 1]
```diff
@@ -32,16 +29,23 @@
                                     {% endfor %}
                                 {% endif %}
                             </div>
-                            <div class=" container-fluid">
+                            <div class="container-fluid three-buttons">
```
Comment (Aymammet) [inline templates/post-edit.html:32]: Why "three-buttons"?
What if we add more button in further steps? will we change the name as "four-buttons"
or
What if we have another three buttons in other templates? will you name them also "three-buttons"?
Use meaningful names. Names that give us the functionality of the button.

**Review by Aymammet — CHANGES_REQUESTED** [review]

File: templates/post-edit.html (hunk 1 above)
Comment (anurmamedov) [reply templates/post-edit.html]: updated changes made. thank you for comments

**Comment by anurmamedov** [comment]
1. three-buttons renamed into postedit-buttons. 
2.  once it gets less than 410px buttons transforms into row position.

image

**Review by Aymammet — APPROVED** [review]

File: templates/post-edit.html (hunk 1 above)
Comment (Aymammet) [reply templates/post-edit.html]: Thank you too, so if it is ok mark this conversations as resolved.
Button is down here

Comment (anurmamedov) [reply templates/post-edit.html]: Done

**Pull request merged**